pom.xml

```diff
@@ -135,21 +135,21 @@
 
 		<!-- Project Versions -->
 		<asm.version>9.5</asm.version>
-		<aspectj.version>1.9.19</aspectj.version>
+		<aspectj.version>1.9.20</aspectj.version>
 		<assertj-core.version>3.24.2</assertj-core.version>
-		<bouncycastle.version>1.74</bouncycastle.version>
-		<byte-buddy.version>1.14.4</byte-buddy.version>
+		<bouncycastle.version>1.76</bouncycastle.version>
+		<byte-buddy.version>1.14.6</byte-buddy.version>
 		<cdi-unit.version>4.1.0</cdi-unit.version>
 		<cglib.version>3.3.0</cglib.version>
 		<commons-collections4.version>4.4</commons-collections4.version>
 		<commons-fileupload.version>1.5</commons-fileupload.version>
-		<commons-io.version>2.12.0</commons-io.version>
-		<commons-lang3.version>3.12.0</commons-lang3.version>
+		<commons-io.version>2.13.0</commons-io.version>
+		<commons-lang3.version>3.13.0</commons-lang3.version>
 		<guice.version>5.1.0</guice.version>
 		<el-impl.version>2.2.1-b05</el-impl.version>
 		<forbiddenapis.version>3.5.1</forbiddenapis.version>
 		<jacoco.version>0.8.7</jacoco.version>
-		<jackson.version>2.14.3</jackson.version>
+		<jackson.version>2.15.2</jackson.version>
```

Comment:
I have a small doubt about this one.

Reply:
What's wrong about it? It works as expected in wicketstuff :)

Reply:
https://issues.apache.org/jira/browse/AVRO-3754
There was a warning from a Jackson committer in the Avro project.

Reply:
Sounds scary
I'll try to revert this one in several projects :(